sam-styles/packages/components/button-group/button-group.scss

```diff
@@ -39,4 +39,44 @@
       @include u-border('secondary-dark');
     }
   }
+}
+
+.sds-button-group--modular.sds-button-group--segmented {
+  mat-button-toggle:not(:last-of-type){
+    margin-right: 10px;
+  }
+  .sds-button-group__item {
+    .mat-button-toggle-button{
+      padding: 8px;
+    }
+    border-radius: 8px;
+    box-shadow: 1px 2px 4px 0px #00000080;
+    background-color: var(--base-lightest, #F9F9F7); 
```

Comment:
Is there any specific reason to use var, can't we use **color("base-lightest")** without specifying the color code

Reply:
From what I remember the mixin did not work when I tried it. Let me double check and I'll update it if it actually does work.

Reply:
@yerramshilpa I just updated these